wordpress-mobile/WordPress-iOS
# Sets the date_created_gmt field when creating a page

The `date_created_gmt` field is used for the Page List sorting so it should be filled when creating a new page.

**Fixes** #7148 

**To test:**

First make sure you can replicate the issue on develop.
Then checkout this branch and check that you can't :)
I am not 100% sure this is the right solution, but it works wonderfully well and is the only thing I can think of doing because the field `date_created_gmt` is what is being used to sort the Page List.

Needs review: @koke
